app/controller/member.controller.js

```diff
@@ -146,3 +147,36 @@ exports.create = async (req, res) => {
     });
 };
 
+
+/*****************************8
+ * Developer: corner
+ * Description: 계정 삭제
+ *             회원의 계정을 삭제합니다.
+ *             params : m_no
+*****************************/
+
+exports.delete = async (req, res) => {
+	if (req.query.hasOwnProperty('m_no') && req.query.m_no === '') {
```

Comment:
Codacy has a fix for the issue: [Replace `'m_no')·&&·req.query.m_no·===·''` with `"m_no")·&&·req.query.m_no·===·""`](https://app.codacy.com/gh/Eight-Corner/oneuleun_web_api/pullRequest?prid=9683172)
```suggestion
	if (req.query.hasOwnProperty("m_no") && req.query.m_no === "") {
```